app/instrumentation-tests/src/org/commcare/androidTests/DemoUserOfflineTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.commcare.androidTests
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import org.commcare.utils.InstrumentationUtility
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+class DemoUserOfflineTest: DemoUserTest() {
+
+    companion object {
+        const val CCZ_NAME = "demo_user_test_1.ccz"
+        const val APP_NAME = "Demo User Restore Test"
+    }
+
+    @Before
+    fun setup() {
+        InstrumentationUtility.changeWifi(false)
+        installApp(APP_NAME, CCZ_NAME, true)
+    }
+
+    @Test
+    fun testPracticeMode_online() {
+        testPracticeMode()
+    }
+
+    @Test
+    fun testPracticeMode_withUpdatedApp_online() {
```

Comment:
this would need to be suppressed below Android 8 right ?

Reply:
Thanks for catching this one. I totally missed it.